rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -228,7 +228,7 @@ def _detect_relevant_trades(
             if receive_event is None:
                 continue
 
-            if swap_data.from_asset != self.native_asset:
+            if swap_data.from_asset not in (self.native_asset, self.wrapped_native_asset):
```

Comment:
I am still not sure about that. Isn't it also possible to just swap wxDAI, or wETH that you have in your wallet via cowswap?

Reply:
checked again and confirmed it's a case of a new eth flow contract being used. adjusted the code accordingly.